pkgs/servers/mastodon/default.nix

```
@@ -0,0 +1,77 @@
+{ nodejs-slim, yarn2nix-moretea, fetchFromGitHub, bundlerEnv,
+  stdenv, yarn, lib, callPackage, ... }:
```

```suggestion
  stdenv, yarn, lib, callPackage, imagemagick, ffmpeg, file ... }:
```
These need to be added here since they are used below.

---

done